### PR TITLE
UX tweaks

### PR DESCRIPTION
**Pull request by ahemphill**

### CODE REVIEW

File: tools/scoring/README.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ To install the dependencies, please run the `python3 setup.py install` from the
 ## Usage
 
 The application can be run from the command line. It takes three arguments:
-1. `-ont/--ontology` (required): Absolute path for the directory which contains your ontology
-2. `-prop/--proposed` (required): Absolute path for your proposed configuration file (to be scored)
-3. `-sol/--solution` (required): Absolute path for your solution configuration file
+1. `-prop/--proposed` (required): Absolute path for your proposed configuration file (to be scored)
+2. `-sol/--solution` (required): Absolute path for your solution configuration file
+3. `-ont/--ontology` (optional): Absolute path for the directory which contains your ontology. Defaults to `ontology/yaml/resources`
```
Comment (charbull) [inline tools/scoring/README.md:16]: can you name it as in the ontology-validator please

File: tools/scoring/score/parse_config.py [hunk 2]
```diff
@@ -82,6 +84,7 @@ def append_types(
         The deserialized files dictionary with types
         appended to each entity in the respective files.
     """
+    print('Scoring — appending entity types')
```
Comment (charbull) [inline tools/scoring/score/parse_config.py:87]: I dont think this is needed? it is an implementation detail?

File: tools/scoring/score/parse_config.py [hunk 3]
```diff
@@ -199,6 +202,7 @@ def aggregate_results(
     results = {}
 
     for dimension in dimensions:
+      print(f'Scoring — calculating {dimension.__name__}')
```
Comment (charbull) [inline tools/scoring/score/parse_config.py:205]: this might be too much verbose?

**Review by charbull — CHANGES_REQUESTED** [review]

File: tools/scoring/score/parse_config.py (hunk 2 above)
Comment (ahemphill) [reply tools/scoring/score/parse_config.py:87]: Fair point.

Comment (ahemphill) [reply tools/scoring/score/parse_config.py:87]: Well, it contextualizes that follows, which _is_ relevant.

**Review by charbull — APPROVED** [review]

**Pull request merged**